src/algorithms/post/election_post.md

```diff
@@ -208,6 +208,6 @@ While it resets a miner’s ElectionWindow, miners earn no reward from submittin
 
 Storage Power Consensus participants are subject to a {{<sref min_miner_size>}}, meaning miners smaller than `MIN_MINER_SIZE_STOR` of active (or in-deal) storage or whose active storage represents less than `MIN_MINER_SIZE_PERC`% of the network's cannot produce valid electionPoSts. 
 
-However, these miners still have active power and can still run and transmit SurprisePosts as messages to be added on-chain. These miners can also be faulted as usual for lacking to prove their power in a proving period. 
+These miners' power does not count as part of the total network power, nor are they able to sumit electionPoSts but they can still run and transmit SurprisePosts as messages to be added on-chain. These miners can also be faulted as usual for lacking to prove their power after `MAX_FAULT_COUNT` challenges. 
```

Comment:
Blocking: These miners can also be faulted as usual *and be terminated* for failing in `MAX_FAULT_COUNT` consecutive epochs.

Reply:
good catch. If you're ok I'll actually simplify by removing max fault count here since it's mentioned in a ton of places.